extensions/amp-next-page/amp-next-page.md

```diff
@@ -163,82 +148,173 @@ Elements which are common across multiple pages can be programmatically
+
+| Key                | Value                                                                                                                                                                |
+| ------------------ | -------------------------------------------------------------------------------------------------------------------------------------------------------------------- |
+| `pages` (required) | Array of pages, exactly the same format as the in the inline configration section above.                                                                             |
+| `next` (optional)  | Optional string url pointing to the next remote to query for more pages (should abide by the same rules as the initial URL, namely implement the CORS requirements.) |
+
+#### Example Remote Configuration
+
+The following configuration returned from the server will recommend two more documents for the user to read and tell `amp-next-page` to query `https://example.com/more-pages` when the user finishes reading the provided recommendations.
 
 ```json
 {
   "pages": [
     {
       "image": "https://example.com/image1.jpg",
       "title": "This article shows first",
-      "ampUrl": "https://example.com/article1.amp.html"
+      "url": "https://example.com/article1.amp.html"
     },
     {
       "image": "https://example.com/image2.jpg",
       "title": "This article shows second",
-      "ampUrl": "https://example.com/article2.amp.html"
+      "url": "https://example.com/article2.amp.html"
     }
   ],
-  "hideSelectors": [".header", ".footer"]
+  "next": "https://example.com/more-pages"
 }
 ```
 
+## Attributes
+
+### `src`
+
+The URL of the remote endpoint that returns the JSON that will be used to
+configure this `amp-next-page` component. This must be a CORS HTTP service.
+The URL's protocol must be HTTPS.
+
+{% call callout('Important', type='caution') %} Your endpoint must implement
+the requirements specified in the CORS Requests in AMP spec. {% endcall %}
+
+The `src` attribute is required unless a config has been specified inline.
+
+### `max-pages` (optional)
+
+The maximum number of pages to load and show to the user. When this number is exceeded, the footer box will be shown with links to the remaining articles. The default is `Infinity`.
+
+### `deep-parsing` (optional)
+
+When specified, this attribute allows `amp-next-page` to recursively parse inline configuration from the loaded documents (if the documents themselves contain an `<amp-next-page>` tag). This behavior is enabled by default when using an inline configuration and disabled by default when using a remote configuration.
+
+### `xssi-prefix` (optional)
+
+When specified, this attribute allows `amp-next-page` to strip a prefix from the fetched JSON before parsing. This can be useful for APIs that include [security prefixes](http://patorjk.com/blog/2013/02/05/crafty-tricks-for-avoiding-xssi/) like `)]}` to help prevent cross site scripting attacks.
```

Comment:
Suggestion:
```
The xssi-prefix attribute enables <amp-next-page> to strip a prefix before parsing remotely hosted JSON.  This can be useful for APIs that include [security prefixes](http://patorjk.com/blog/2013/02/05/crafty-tricks-for-avoiding-xssi/) like `)]}` to help prevent cross site scripting attacks.
```